tests/core/test_common.py

```diff
@@ -185,6 +186,38 @@ def test_collector(self):
             tag = "check:%s" % check.name
             assert tag in all_tags, all_tags
 
+
+    @attr(requires='sysstat')
+    def test_apptags(self):
+        '''
+        Tests that the app tags are sent if specified so 
+        '''
+        agentConfig = {
+            'api_key': 'test_apikey',
+            'collect_ec2_tags': False,
+            'collect_instance_metadata': False,
+            'create_dd_check_tags': True,
+            'version': 'test',
+            'tags': '',
+        }
+
+        # Run a single checks.d check as part of the collector.
+        redis_config = {
+            "init_config": {},
+            "instances": [{"host": "localhost", "port": 6379}]
+        }
+        checks = [load_check('redisdb', redis_config, agentConfig)]
+
+        c = Collector(agentConfig, [], {}, get_hostname(agentConfig))
+        payload = c.run({
+            'initialized_checks': checks,
+            'init_failed_checks': {}
+        })
+
+        # We check that the redis DD_CHECK_TAG is sent in the payload
+        assert DD_CHECK_TAG.format('redisdb') in payload['host-tags']['system']
```

Comment:
I may be wrong but I think it's the same : DD_CHECK_TG='dd_check:{0}'. The advantage is that the tag prefixed is defined only at one place (in collector.py). 

I mostly did this because at the time the tag prefix wasn't officially defined and I knew I would forget to update the test if I had to change it. But I can replace that with dd_check:redisdb if you think it's important to check that we didn't change the apptag prefix by accident in our test suite. What should I do ?